openml-h2o/src/test/java/com/feedzai/openml/h2o/H2OModelProviderTrainTest.java

```diff
@@ -120,9 +157,29 @@ protected Dataset getTrainDataset() {
         builder.put(H2OAlgorithm.NAIVE_BAYES_CLASSIFIER, H2OAlgorithmTestParams.getBayes());
         builder.put(H2OAlgorithm.XG_BOOST, H2OAlgorithmTestParams.getXgboost());
         builder.put(H2OAlgorithm.GENERALIZED_LINEAR_MODEL, H2OAlgorithmTestParams.getGlm());
+        builder.put(H2OAlgorithm.ISOLATION_FOREST, H2OAlgorithmTestParams.getIsolationForest());
 
         return builder.build();
     }
 
+    /**
+     * Tests that the model loaded by {@link H2OModelCreator#fit(Dataset, Random, Map)} is able to score instances, based on a schema
+     * with no target variable.
+     */
+    @Test
+    public final void testIsolationForestWithDatasetWithoutTargetVariable() throws ModelTrainingException {
+        final H2OModelCreator loader = getMachineLearningModelLoader(H2OAlgorithm.ISOLATION_FOREST);
+        final Map<String, String> params = H2OAlgorithmTestParams.getIsolationForest();
+
+        final Random random = new Random(234);
+        final Dataset dataset = createDataset(SCHEMA_NO_TARGET_VARIABLE);
+
+        final ClassificationH2OModel model = loader.fit(dataset, random, params);
+
+        final MockInstance dummyInstance = new MockInstance(dataset.getSchema(), random);
+        assertThatCode(() -> model.getClassDistribution(dummyInstance))
+                .as("Scoring instance '%s' succeeds", dummyInstance)
+                .doesNotThrowAnyException();
```

Comment:
Please make an assertion that the scores sum to 1.0 (we have this in other places of the code)

Also test the "getClassification"